.evergreen/config.yml

```diff
@@ -0,0 +1,655 @@
+        variables:
+           VERSION: "3.0"
+
+  - id: edge-versions
+    display_name: MongoDB Version
+    values:
+      - id: "latest-stable"
+        display_name: "4.4"
+        variables:
+          VERSION: "4.4"
+      - id: "oldest-supported"
+        display_name: "3.0"
+        variables:
+          VERSION: "3.0"
+
+  - id: driver-versions
+    display_name: Driver Version
+    values:
+      - id: "latest-stable"
+        display_name: "1.8.0beta2"
+        variables:
+          DRIVER_VERSION: "1.8.0beta2"
+      - id: "latest-minor-dev"
+        display_name: "1.8-dev"
+        variables:
+          DRIVER_BRANCH: "v1.8"
+      - id: "latest-dev"
+        display_name: "1.9-dev (master)"
+        variables:
+          DRIVER_BRANCH: "master"
+
+  - id: os-php7
+    display_name: OS
+    values:
+      - id: debian92-test
+        display_name: "Debian 9.2"
+        run_on: debian92-test
+      - id: rhel70-test
+        display_name: "RHEL 7.0"
+        run_on: rhel70
+      - id: rhel71-power8
+        display_name: "RHEL 7.1 Power 8"
+        run_on: rhel71-power8-test
+      - id: rhel74-zseries
+        display_name: "RHEL 7.4 zSeries"
+        run_on: rhel72-zseries-test
+      - id: ubuntu1804-arm64-test
+        display_name: "Ubuntu 18.04 ARM64"
+        run_on: ubuntu1804-arm64-test
+
+  - id: topology
+    display_name: Topology
+    values:
+      - id: standalone
+        display_name: Standalone
+        variables:
+           TOPOLOGY: "server"
+      - id: replicaset
+        display_name: Replica Set
+        variables:
+           TOPOLOGY: "replica_set"
+      - id: sharded-cluster
+        display_name: Sharded Cluster
+        variables:
+           TOPOLOGY: "sharded_cluster"
+
+  - id: auth
+    display_name: Authentication
+    values:
+      - id: auth
+        display_name: Auth
+        variables:
+           AUTH: "auth"
+      - id: noauth
+        display_name: NoAuth
+        variables:
+           AUTH: "noauth"
+
+  - id: ssl
+    display_name: SSL
+    values:
+      - id: ssl
+        display_name: SSL
+        variables:
+           SSL: "ssl"
+      - id: nossl
+        display_name: NoSSL
+        variables:
+           SSL: "nossl"
+
+  - id: storage-engine
+    display_name: Storage
+    values:
+      - id: mmapv1
+        display_name: MMAPv1
+        variables:
+           STORAGE_ENGINE: "mmapv1"
+      - id: wiredtiger
+        display_name: WiredTiger
+        variables:
+           STORAGE_ENGINE: "wiredtiger"
+      - id: inmemory
+        display_name: InMemory
+        variables:
+           STORAGE_ENGINE: "inmemory"
+
+
+buildvariants:
+
+- matrix_name: "test-baseline"
+  matrix_spec: {"os-php7": ["debian92-test", "ubuntu1804-arm64-test", "rhel70-test"], "php-versions": "*", "versions": "4.4", "driver-versions": "latest-stable" }
+  exclude_spec:
+    - { "os-php7": ["debian92-test", "rhel70-test"], "php-versions": "7.0", versions: "*", "driver-versions": "*" }
```

Comment:
Is `versions: "*"` overkill since the matrix is only testing 4.4? Or is this just to be defensive in case `matrix_spec` is changed? I assume it may be the latter as you did the same for `driver-version` here.

Reply:
Purely being defensive here as I noticed 7.0 is not available on these platforms, regardless of server version.